include/vcpkg/base/messages.h

```diff
@@ -993,9 +1012,17 @@ namespace vcpkg
     DECLARE_MESSAGE(MsiexecFailedToExtract,
                     (msg::path, msg::exit_code),
                     "",
-                    "msiexec failed while extracting '{path}' with launch or exit code {exit_code} and message:");
+                    "msiexec failed while extracting \"{path}\" with launch or exit code {exit_code} and message:");
     DECLARE_MESSAGE(NoLocalizationForMessages, (), "", "No localized messages for the following: ");
     DECLARE_MESSAGE(NoRegistryForPort, (msg::package_name), "", "no registry configured for port {package_name}");
+    DECLARE_MESSAGE(NugetPackageCreationFailed,
+                    (msg::error),
+                    "'{error}' is the NuGet output message.",
+                    "NuGet package creation failed: '{error}' ");
+    DECLARE_MESSAGE(NugetPackageFileCreationFailed,
+                    (msg::path),
+                    "",
+                    "NuGet package creation failed. No .nupkg was produced. \"{path}\" ");
```

Comment:
```suggestion
    DECLARE_MESSAGE(NugetPackageFileSucceededButCreationFailed,
                    (msg::path),
                    "",
                    "NuGet package creation succeeded, but no .nupkg was produced. Expected: \"{path}\"");
```

I think this change lost the impact of the original product code's meaning that we got a zero exit code from running `nuget pack` but there was no `nupkg` for some reason; this may prompt the user to take different troubleshooting steps or that sort of thing.